apps/andi/lib/andi_web/live/ingestion_live_view/metadata_form.ex

```diff
@@ -65,6 +81,13 @@ defmodule AndiWeb.IngestionLiveView.MetadataForm do
     {:noreply, socket}
   end
 
+  def handle_info(
+        %{topic: "ingestion-published"},
+        socket
```

Comment:
Is this event/topic needed? It looks like `ingestion_published? = ingestion.submissionStatus == :published` does the logic. I'm not seeing where socket.ingestion_published is used

Reply:
Yes that `ingestion_published? = ingestion.submissionStatus == :published` in the mount method determines the initial `ing_published?` state. The mount method is never called again though, despite the parent state changing.

- https://github.com/phoenixframework/phoenix_live_view/issues/396
- https://github.com/UrbanOS-Public/internal/blob/main/wiki_files/live_view_notes.md

This `handle_info` method responds to a successfully publish and updates that attribute in response.

Open to ideas on how to make that more clear. Imo it's just remembering that the mount only happens once.